### PR TITLE
Fix all checkstyle issues.

### PR DESCRIPTION
**Pull request by MaxGroenenboom**
Fixed all checkstyle issues.
Fixed all warnings.
Changed indentation for `Function`, `ParameterList` and other lists of `Parameter` constructions to enhance readability.

Travis: [Build Status](https://travis-ci.org/Denpeer/tygron)

### CODE REVIEW

**Comment by JoshCode** [comment]
Build passes, so approved.

\- CodeFox [CF](http://github.com/CodeFoxNL)

**Comment by HarmGriffioen** [comment]
Merge conflicts, please resolve them

**Comment by MaxGroenenboom** [comment]
Yeah those were introduced by merging the other PR's...

**Comment by MaxGroenenboom** [comment]
So I got a failing test now: `tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8` (with a typo in the name of the test btw), while I obviously didn't change any functional code...
Also I did test locally and it succeeded :/

File: contextvh/src/main/java/contextvh/translators/J2UpgradeType.java [hunk 1]
```diff
@@ -18,6 +18,13 @@
 	private static Translator translator = Translator.getInstance();
 
 	/**
+	 * Adds a translator for J2UpgradePair if it isn't added yet.
+	 */
+	public J2UpgradeType() {
+		translator.registerJava2ParameterTranslator(new J2UpgradePair());
```
Comment (MaxGroenenboom) [inline contextvh/src/main/java/contextvh/translators/J2UpgradeType.java]: Please verify this is necessary, I haven't seen this in other Translator classes that use other Translators, but for some reason the test for this class was failing.

`tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8`

**Comment by MaxGroenenboom** [comment]
Please verify f6816f1 is necessary, I haven't seen this construction in other Translator classes that use other Translators, but for some reason the test for this class was failing.

`tranlatorTest1(contextvh.translators.J2UpgradeTypeTest): No translator found for class nl.tytech.data.engine.serializable.UpgradePair$$EnhancerByMockitoWithCGLIB$$75fc3dd8`

**Comment by MaxGroenenboom** [comment]
Travis builds now so I hope this can be reviewed and merged quickly.

**Comment by levilime** [comment]
So this is a refactor because of the checkstyle change in #87? A member of team Danshal was okay with not having any trailing spaces so we'll see this as part of that merge too  and agree with doing away with all spaces in the code. Also the added comments is a nice touch.

:star:

**Comment by MaxGroenenboom** [comment]
Yeah I figured it was now possible to just checkstyle the entire module to make it easier to check for checkstyle issues.

**Comment by spiderbiggen** [comment]
@MaxGroenenboom for the problem with J2UpgradeType test it seems that test is also missing the 3rd test. It seems the changes to that test were lost in a merge

**Comment by MaxGroenenboom** [comment]
@spiderbiggen I didn't remove it as you can see in the diff...
https://github.com/eishub/tygron/pull/93/files#diff-bde1517c676bee363d4351b115a5b107L16

**Comment by spiderbiggen** [comment]
I know but it is lost somewhere in a merge

**Comment by spiderbiggen** [comment]
ah found it

**Comment by MaxGroenenboom** [comment]
@spiderbiggen according to this history (https://github.com/eishub/tygron/commits/context/contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java) it was removed by a revert of a PR from @klikooo. If I remember right he reverted a PR that caused the context branch to fail on travis.

**Comment by MaxGroenenboom** [comment]
Here is the revert PR I mentioned in the last comment: #89

**Comment by spiderbiggen** [comment]
I will create a fix

**Comment by spiderbiggen** [comment]
all right I will do that after this is merged as it now always breaks on checkstyle
we approve of this PR

RAMpestampers

**Pull request merged**